### DISCUSSION TITLE
Aun es necesario https://launchxauth.azurewebsites.net/joingh?

### DISCUSSION THREAD

**Discussion by KevAguirre** [Launch-X-Latam/MisionFrontEnd#3] [General]
Hola Comandante, aun es necesario hacer este fill-out:
https://launchxauth.azurewebsites.net/joingh
?

**Comment by OscarNoe** [comment]
No mientras el repositorio este como publico

**Comment by GalaxyStyle1** [comment]
entonces como hacemos por ahora esperamos  ? y aplicamos mañana comander

**Reply by UncannyChimp** [reply]
Se supone que ya con lo que aplicamos estaria, pero lo mejor seria estar activo en instagram por si se notifica de nuevo que se abra lo de git hub

**Reply by GalaxyStyle1** [reply]
gracias estare pendiente

**Comment by Emilio17MC** [comment]
Es muy probable que si sea necesario, en la edición pasada llegaron invitaciones a repositorios privados de la organización para continuar en la misión backend. Además de que ayuda a demostrar que eres parte de LaunchX en tu perfil 🙌.

De momento solo queda esperar a que esté nuevamente disponible el registro para que te llegue la invitación a la organización.

**Comment by BotLX834** [comment]
Hola. Hay que seguir intentando la validacion por que en la bienvenida se menciono que el repositorio no siempre sera publico. Yo estuve intentando la validacion desde el viernes 23 y hasta hoy martes 27 volvi a intentar y ya me llego la invitacion a mi correo de inovaccion.
Si este repositorio o algun otro de los modulos que siguen es privado ya no podran ver su contenido sin la invitacion.
